### PR TITLE
Add screen-studio extension

### PR DESCRIPTION
**Pull request by pie6k**
## Description

This extension allows controlling Screen Studio (https://screen.studio) app from Raycast easily.

It allows recording a window, entire display or selected area. It also allows finishing, restarting and cancelling the recording in progress.

## Screencast

CleanShot 2023-05-19 at 15 02 46

Note; the extension on screencast is duplicated (one is local, dev version of it)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pie6k** [comment]
Check failed with 

```
Error:   9:11  error  Invalid owner "screen-studio". Use the Raycast organization handle
```

I'm not sure why is that. The handle is correct.

Could it because by accident I first published it without "access": "public" flag so it was instantly published as an internal extension, and only then I modified it to public, which created this PR?

**Comment by peduarte** [comment]
@pie6k Hey man, thanks for submitting this extension! The build failed because only public organisations can publish extensions to the public store — but by default extensions are private. I've now made it public, and the build passes ✅ 

I'm going to start reviewing your extension now.

File: extensions/screen-studio/src/restart-recording.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/restart-recording.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/record-window.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/record-window.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/record-display.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/record-display.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/record-area.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/record-area.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/finish-recording.tsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/finish-recording.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/cancel-recording.tsx [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import { ActionPanel, Action, Icon, List, open } from "@raycast/api";
```
Comment (peduarte) [inline extensions/screen-studio/src/cancel-recording.tsx:1]: Nit: Remove unused imports, `ActionPanel`, `Action`, `Icon`, and `List`

File: extensions/screen-studio/src/record-area.tsx [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+
+export default async function Command() {
+  await open("screen-studio://record-area");
```
Comment (peduarte) [inline extensions/screen-studio/src/record-area.tsx:4]: Feature request for the future 😇 One thing I'd personally really like is to be able to save specific areas, so I could easily start an area recording from a saved preset

```
open("screen-studio://record-area?w=500&h=800&x=0&y=0")
```

Then this extension could let users create them and easily start a recording via a Hotkey.

Could also be implemented directly in Screen Studio and exposed in Raycast

**Review by peduarte — APPROVED** [review]
Very nice 💖

Love how simple, yet helpful, this extension is, thanks to the specific deeplinks you created.

File: extensions/screen-studio/src/record-area.tsx (hunk 7 above)
Comment (pie6k) [reply extensions/screen-studio/src/record-area.tsx:4]: I agree, but I think I'll add it at later time. I'll need to first figure out some good UX around it in the app itself.

**Comment by pie6k** [comment]
@peduarte Thank you! I fixed all Lint errors you mentioned. I'll add more features to it (such as custom areas), but for now I'd like to release "MVP" version of it

**Review by peduarte — APPROVED** [review]

**Pull request merged**